add definition of a VO (#41)
WLCG-AuthZ-WG/common-jwt-profile: profile.md

# Introduction

This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the virtual organisation, applications and relying parties.

A virtual organisation (VO) is a group of users that are

> self-managing, often with internal structure (such as groups and

Reviewer: ```suggestion
independently managed, often with internal structure (such as groups and
```
At Fermilab we have a group that provides services to many VOs (both those that we consider sub-VOs and full VOs), including the management of computing infrastructures.  So `self-managed` isn't accurate for them.

Author: Thanks.

roles).  The members of a VO work towards a common goal; for example,
maximising the scientific output from a specific scientific

> instrument.  A VO has well understood membership criteria, which the

Reviewer: ```suggestion
instrument  or providing shared services to multiple related projects.
A VO has well understood membership criteria, which the
```

Author: Thanks for the suggestion.  I've updated the text.

I was wondering if you have a good example of a service-providing VO, that I include in the list of example VOs?

Reviewer: `fermilab` and `osg` are the two that I know about.

Author: Thanks

members of the VO manage themselves.  Their work is often supported

> through the use of resources located multiple facilities.  Within

Reviewer: ```suggestion
through the use of resources located at multiple facilities.  Within
```
